packages/nvidia_nat_opentelemetry/src/nat/plugins/opentelemetry/register.py

```diff
@@ -192,3 +192,41 @@ async def galileo_telemetry_exporter(config: GalileoTelemetryExporter, builder:
         drop_on_overflow=config.drop_on_overflow,
         shutdown_timeout=config.shutdown_timeout,
     )
+
+
+class DBNLTelemetryExporter(BatchConfigMixin, TelemetryExporterBaseConfig, name="dbnl"):
+    """A telemetry exporter to transmit traces to DBNL."""
+
+    endpoint: str = Field(description="The DBNL OTEL traces endpoint.", default="")
```

Comment:
why not just accept a base URL and compute the endpoint always? It is more confusing for users to remember an exact endpoint spec `/api/otel/v1/traces` compared to just remember host + port